pyproject.toml

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2025, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
+[project]
+name = "setup_ovs"
+version = "1.0"
+description = "Apply an OVS configuration from a JSON file"
+authors = [
+    { name = "RTE", email = "mathieu.dupre@savoirfairelinux.com" }
```

Comment:
Not sure it was the intended translation from setup.py (for vm_manager, I gave 2 authors, RTE (no email) and Mathieu (with his email))